custom-example/android/AndroidManifest.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0"?>
+<manifest package="org.custom.qgroundcontrol" xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="-- %%INSERT_VERSION_NAME%% --" android:versionCode="-- %%INSERT_VERSION_CODE%% --" android:installLocation="auto">
```

Comment:
Please note `INSERT_VERSION_CODE` and `INSERT_VERSION_NAME` that [allegedly](http://www.cutehacks.com/blog/2015/5/28/one-version-number-to-rule-them-all) appeared in Qt5.2  (I tested them to be there on Qt.5.12.5, while QGC is now on Qt.5.12.6) 

They work against:
```
+ANDROID_VERSION_NAME="1.2.3"
+ANDROID_VERSION_CODE=123
```
Which I am not setting here, but could write some script that automatically counts the commits for the `..code` and replicates `$${CUSTOM_QGC_VERSION}` for the `..name`.